src/protocol/requests/deleteGroups/index.js

```diff
@@ -0,0 +1,17 @@
+const versions = {
+  0: groupIds => {
+    const request = require('./v0/request')
+    const response = require('./v0/response')
+    return { request: request(groupIds), response }
+  },
+  1: groupIds => {
```

Comment:
Let's implement v2 as well, since it's almost the same as v1 (just compact strings instead of strings)

Reply:
Implemented

Reply:
It looks like you forgot to add it to this file.